assets/src/components/nav/mobilePortraitNav.tsx

```diff
@@ -25,6 +25,8 @@ const MobilePortraitNav = ({
 
   return (
     <div className="m-nav--narrow">
+      <div className="m-nav__app-content">{children}</div>
```

Comment:
question (non-blocking): There may be an issue with how I'm testing this since chrome doesn't seem to let you resize down to "mobile" sizes, but when I switch to responsive mode in the inspector tools and go from desktop => mobile, the state is lost. Is this an artifact of how I'm testing? Though maybe not an issue if it is impossible for users to resize their desktop windows that small anyway!

Reply:
Interesting - in Firefox you can resize the window narrow enough to hit the mobile breakpoint and I still see the state resetting. I think there must be some combination of the React element tree and the DOM at play here, but I think I can refactor things a bit more in order to get it all to work.

Reply:
I think that f0bcf458 should fix this, though I'm still not sure if the process you described in Chrome dev tools will preserve the state or not.

Reply:
Yes, I'm not seeing the state reset anymore in chrome either! 🎉